src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  margin-top: 0;
+}
+
+.form {
+  position: relative;
+  width: 100%;
+  margin-top: 20px;
+  display: inline-block;
+}
+
+.form-label {
+  display: inline-block;
+  width: 100%;
+}
+
+.top-search-bar {
+  height: 70px;
+  line-height: 70px;
+}
+
+.bottom-search-bar {
+  height: 42px;
+  line-height: 42px;
+}
+
+.search-image {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.big-search-image {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+}
+
+.small-search-image {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+}
+
+.input-field {
+  outline: none;
```

Comment:
@iLebo-stack what about the text color?
it should be like in the mock - take a closer look at `color` for input